js/package.json

```
@@ -32,7 +32,7 @@
     "postinstall": "patch-package"
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.0.0-rc.4",
+    "@jupyterlab/builder": "^4.0.0",
```

I'm guessing this should also allow for version 3

```suggestion
    "@jupyterlab/builder": "^3.0.0-rc.4 || ^4.0.0",
```

---

`from versions ... 3.2.8, 3.2.9, 3.3.0a1, 4.0.0a0, 4.0.0a1, ...`
so 3.2.9 is the latest available non-alpha version. If it still works...